src/ICal/Event.php

```diff
@@ -43,6 +43,20 @@ class Event
      */
     public $dtstamp;
 
+    /**
+     * When the event starts, represented as a timezone-adjusted string
+     *
+     * @var string $dtstart
+     */
+    public $dtstart_tz;
+
+    /**
+     * When the event ends, represented as a timezone-adjusted string
+     *
+     * @var string $dtend_tz
```

Comment:
`@var $dtend_tz`?

Reply:
Are you wanting to leave out the type? I would have thought most users would expect this to be declared (although obviously in the docblock, given that this package has to support PHP5.3)...

Reply:
Less about the PHP 5.3 aspect but more that all other docblocks in this file don't specify a type.

Reply:
As this is one of very few dedicated parsers for PHP out there, it would be good to modernise it over time (stronger typing, etc). If that's desirable then I guess the choice is whether to do that incrementally, or in larger batches. 
Either way, happy to remove the types if you'd prefer?

Reply:
Yep not against modernisation but at this point in time I'd prefer consistency.

Reply:
Ok, no problem.